src/utils/hooks/useKubevirtHyperconvergeConfiguration.ts/index.ts

```diff
@@ -0,0 +1,22 @@
+import { V1KubeVirtConfiguration } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { KUBEVIRT_HYPERCONVERGED, OPENSHIFT_CNV } from '@kubevirt-utils/constants/constants';
+import { isUpstream } from '@kubevirt-utils/utils/utils';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+
+import { KUBEVIRT_HC_GROUP_VERSION_KIND, KUBEVIRT_HC_NAME } from './constants';
+
+export type KubevirtHyperconverged = K8sResourceCommon & {
+  spec: {
+    configuration: V1KubeVirtConfiguration;
+  };
+};
+
+const useKubevirtHyperconvergeConfiguration = () => {
```

Comment:
Only 1 component uses it? 
if the answer is yes, there is no need for this hook as it does no more the k8swatch, no logic, so use the k8swatch hook
if the answer is no, please amend it a bit to be nicer. Don't simply return the k8swatch hook itself; do the same as we do in all our other hooks.